### PR TITLE
Fix Streaming Integration Test Failure in Precommit

### PR DESCRIPTION
**Pull request by markflyhigh**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

Made some changes in profiles which is used to start integration tests in Jenkins pre-commit.

Instead of having two profiles for batch and streaming integration tests separately, specify integration tests in execution part of each runner.

No change is needed on Jenkins side.

### CODE REVIEW

File: examples/java/pom.xml [hunk 1]
```diff
@@ -138,12 +137,17 @@
                   <goal>verify</goal>
                 </goals>
                 <configuration>
+                  <includes>
+                    <include>WordCountIT.java</include>
```
Comment (lukecwik) [inline examples/java/pom.xml]: Based upon the maven documentation, it is unclear whether this will work without it being prefixed by **/
Did you try this out?

File: examples/java/pom.xml [hunk 2]
```diff
@@ -138,12 +137,17 @@
...
+                    <include>WindowedWordCountIT.java</include>
+                  </includes>
                   <parallel>all</parallel>
                   <threadCount>4</threadCount>
                   <systemPropertyVariables>
                     <beamTestPipelineOptions>
                       [
                       "--project=apache-beam-testing",
+                      "--tempLocation=gs://temp-storage-for-end-to-end-tests",
```
Comment (lukecwik) [inline examples/java/pom.xml]: Why was this needed?

**Comment by lukecwik** [comment]
R: @lukecwik

File: examples/java/pom.xml (hunk 1 above)
Comment (markflyhigh) [inline examples/java/pom.xml]: Yes, I tested it in my own directory. Both "WordCountIT.java" and "**/WordCountIT.java" works.

Comment (lukecwik) [inline examples/java/pom.xml]: Sounds good, it seems simpler without the **/ so you should keep as is.

File: examples/java/pom.xml (hunk 2 above)
Comment (markflyhigh) [inline examples/java/pom.xml]: "--tempLocation" is required for BigQuery I/O.

Test runners have ability to build a temp location based on "--tempRoot", like [this in TestDataflowRunner](https://github.com/apache/incubator-beam/blob/master/runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/testing/TestDataflowRunner.java#L81). But DirecRunner doesn't. So need to specify it separately. 

Not sure why we want a new temp directory parameter in test options. If it's duplicated, I'd like to refactor in another PR.

Comment (markflyhigh) [inline examples/java/pom.xml]: ping @lukecwik

Comment (lukecwik) [inline examples/java/pom.xml]: It seems because we should really be using a Test version of the DirectRunner or it should also  honor the --tempRoot concept like all the other test runners. As you suggest, we should fix this in another PR.

**Comment by lukecwik** [comment]
LGTM
